tests/net/tcp/testcase.yaml

```diff
@@ -1,4 +1,4 @@
 tests:
-  net.tcp:
+  net.tcp.tcp:
```

Comment:
Here would prefer `net.tcp.simple` as we will have enhanced tests for TCP2 in the future.